apps/verticalhome/js/app.js

```diff
@@ -371,6 +377,9 @@
         // receives a home button press.
         case 'hashchange':
           // The group editor UI will be hidden by itself so returning...
+
```

Comment:
I wouldn't squeeze new line between comment and code to which this comment belongs